docs/network-flow-visibility.md

```diff
@@ -678,26 +678,21 @@ spec:
       clickhouse_operator/networks/ip: "::/0"
 ```
 
-ClickHouse credentials are also specified in [flow-aggregator.yml][flow_aggregator_manifest_yaml]
-as a resource of kind: a Secret. Please also make the corresponding changes.
+ClickHouse credentials are also specified in `flow-aggregator.yml` as a resource
+of name: `clickhouse-secret`. Please also make the corresponding changes.
```

Comment:
It will be nice to show these lines of code to help users locate where to change these secrets in `flow-aggregator.yml`
https://github.com/antrea-io/antrea/blob/main/build/yamls/flow-aggregator.yml#L240-L251